src/test/kotlin/lotto/filter/LottoFilterTest.kt

```diff
@@ -0,0 +1,21 @@
+package lotto.filter
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.assertThrows
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ValueSource
+
+class LottoFilterTest {
+
+    @ParameterizedTest
+    @ValueSource(strings = ["1", "10", "21", "30", "41", "45"])
+    fun `들어오는 로또 번호 값이 로또 범위 (1 ~ 45 )  안에 있는 있는지 테스트 한다 통과하면 그대로 값 리턴 된다`(number: Int) {
```

Comment:
```suggestion
    fun `들어오는 로또 번호 값이 로또 범위 (1 ~ 45 )  안에 있는 있는지 확인합니다`(number: Int) {
```
테스트 함수명이 간결하면 좀 더 좋을 것 같아요